code/modules/events/ion_storm.dm

```diff
@@ -502,61 +504,131 @@
 	var/random_player = "The Captain"
 	if(players.len)
 		random_player = pick(players)		//Random player's name, to be used in laws.
+
+	var/random_department = "Science"
+	var/list/departments = list(	"Science",
+									"Supply",
+									"Command",
+									"Engineering",
+									"Security",
+									"Service",
+									"Medical",
+									"Silicon",
+									"Robotics",
+									"Mining",
+									"Botany",
+									"Clown")
+	random_department = pick(departments)	//Random department, to be used in laws.
+
 	var/list/laws = list(	"You are a mouse.",
```

Comment:
```suggestion
	var/static/list/laws = list(	"You are a mouse.",
```

Reply:
By making this variable static, it fails to work due to some statements within changing with random chance.